components/top-nav/src/index.js

```diff
@@ -157,7 +157,7 @@ class TopNav extends Component {
                     <MenuButton open={this.state.navigationOpen} onClick={this.toggleNavigationOpen} />
                   </MenuButtonWrapper>
                   {/* Referenced in MenuButton */}
-                  <Input id="govuk-react-menu-button" type="checkbox" checked={this.state.navigationOpen} />
+                  <Input id="govuk-react-menu-button" type="checkbox" defaultChecked={this.state.navigationOpen} />
```

Comment:
this change fixes an error shown in the console - which suggested either adding an `onClick` or changing this prop

i'm not entirely sure this is the right thing to do here though - as i don't understand native react form handling well enough - @penx thoughts?

Reply:
hmm no I wouldn't change this to defaultChecked as pretty sure that would change the intended behaviour